lib/diff/hex/hex.ex

```diff
@@ -100,15 +109,14 @@ defmodule Diff.Hex do
            "diff",
            "--no-index",
            "--no-color",
+           "--output=#{path_out}",
+           # "--ignore-space-at-eol",
+           # "--ignore-space-change",
```

Comment:
are we introducing these? I know we discussed it before

Reply:
No, not yet at least. Forgot to remove after I was testing them.